### ISSUE TITLE
Whitened gradient descent, a new updating method for optimizers in deep neural networks

### ISSUE THREAD

**Issue by Hiroki11x** [Hiroki11x/Papers#281]
### 論文リンク
https://jad.shahroodut.ac.ir/article_2496.html

## 概要
オプティマイザは、ディープニューラルネットワークにおいて、重みの更新を行う重要なコンポーネントである。本論文では、勾配降下法に基づくオプティマイザーの新しい更新方法である、白化勾配降下法（WGD）を紹介します。この方法は実装が簡単で、勾配降下アルゴリズムに基づくすべてのオプティマイザで使用することができます。また、ネットワークの学習時間を大幅に増加させることもありません。この方法は学習曲線を滑らかにし、分類指標を向上させる。提案アルゴリズムを評価するために、Cifar100とAnimals-10という2つのデータセットに対して、densenet121、resnet18、resnet50という3つのネットワーク構造を用いて48種類のテストを実施した。実験の結果、gradient descentベースのオプティマイザにWGD法を用いることで、分類結果が大幅に改善されることがわかった。例えば、RAdamオプティマイザにWGDを組み込むことで、Animals-10データセットにおいてDenseNetの精度が87.69%から90.02%に向上しました。

## 実験方法・結果

<img width="492" alt="Screenshot 2022-08-23 at 9 58 29 AM" src="https://user-images.githubusercontent.com/8721858/186177752-ce2ba773-8bca-4108-974b-2a154bd6d697.png">

## コメント
論文の質が良くない感じがする
